app/javascript/src/utilities/toast.ts

```diff
@@ -0,0 +1,119 @@
+// import { Wrapper as Flash } from 'Common/components/Flash'
+
+const TIMEOUT = 5000
+
+export type TYPE = 'danger' | 'default' | 'info' | 'success' | 'warning'
+
+const TYPES = ['danger', 'default', 'info', 'success', 'warning']
+
+const typeIcons: Record<string, string> = {
+  danger: 'exclamation-circle',
+  info: 'info-circle',
+  success: 'check-circle',
+  warning: 'exclamation-triangle',
+  default: 'bell'
+}
+
+function createAlertGroupItem (message: string, type = 'default'): HTMLLIElement {
+  const li = document.createElement('li')
+  li.className = 'pf-c-alert-group__item animate__animated animate__slideInRight animate__faster'
+
+  const alertDiv = document.createElement('div')
+  alertDiv.className = `pf-c-alert pf-m-${type}`
+  alertDiv.setAttribute('aria-label', `${type} alert`)
+
+  const iconDiv = document.createElement('div')
+  iconDiv.className = 'pf-c-alert__icon'
+
+  const icon = document.createElement('i')
+  icon.className = `fas fa-fw fa-${typeIcons[type] || 'bell'}`
+  icon.setAttribute('aria-hidden', 'true')
+
+  const titleParagraph = document.createElement('p')
+  titleParagraph.className = 'pf-c-alert__title'
+
+  const screenReaderSpan = document.createElement('span')
+  screenReaderSpan.className = 'pf-screen-reader'
+  screenReaderSpan.textContent = `${type} alert: `
+
+  const actionDiv = document.createElement('div')
+  actionDiv.className = 'pf-c-alert__action'
+
+  const closeButton = document.createElement('button')
+  closeButton.className = 'pf-c-button pf-m-plain'
+  closeButton.type = 'button'
+  closeButton.title = 'Close alert'
+
+  closeButton.onclick = () => { hideToast(li) }
+
+  const closeIcon = document.createElement('i')
+  closeIcon.className = 'fas fa-times'
+  closeIcon.ariaHidden = 'true'
+
+  const titleText = document.createTextNode(message)
+
+  iconDiv.appendChild(icon)
+  titleParagraph.appendChild(screenReaderSpan)
+  titleParagraph.appendChild(titleText)
+  closeButton.appendChild(closeIcon)
+  actionDiv.appendChild(closeButton)
+  alertDiv.appendChild(iconDiv)
+  alertDiv.appendChild(titleParagraph)
+  alertDiv.appendChild(actionDiv)
+  li.appendChild(alertDiv)
+
+  return li
+}
+
+export const showToast = (message: string, type: TYPE = 'default'): void => {
```

Comment:
I think it would be better to implement something more compatible to what we had before, like `toast.notice()` and `toast.error()`, rather than having to specify the type `danger` or `success` on every single call. `danger` and `success` should be constants declared in this file, so we would have a single point to modify if we want to update those types in the future. Also, Imitating the flash API would also make the migration much easier.

If you don't want to change it because it's already done and that would require too much work, I'm fine.

Reply:
On one hand, I would agree, but on the other hand I think we need to forget the "old" semantics ("notice", "error"), and just start thinking in terms of the new one. 

Which does not mean we can't have something like `toast.success(message)`, `toast.danger(message)` etc. This could be actually less error prone maybe. But I think passing the type as an argument is good enough, so I'd keep it as it is, I guess.